certified-connectors/Celonis/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1484 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "EMS API to get Knowledge Models, records, KPIs and filters",
+    "title": "Swagger Specification",
```

Comment:
Title should be the connector name,  please refer this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json